tests/utils.go

```diff
@@ -2096,6 +2076,14 @@ func CreateVmiOnNodeLabeled(vmi *v1.VirtualMachineInstance, nodeLabel, labelValu
 			},
 		},
 	}
+}
+
+// CreateVmiOnNodeLabeled creates a VMI a node that has a give label set to a given value
```

Comment:
I know this code has been there before this PR, but I couldn't make any sense of this comment before reading the function it's supposed to explain, could we remove/rephrase it?
Furthermore, WDYT about renaming the function to  `CreateVmiOnNodeWithLabel`?

Reply:
I would go even further and remove this function as it is pretty easy to achieve it on its own and the function is used in 2 places. But I think this can come in another PR.

Reply:
I agree with @xpivarc, it is a different topic/another PR, let us not hold this nice cleanup ;)